src/nvim/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ set(LINT_SUPPRESS_FILE ${PROJECT_BINARY_DIR}/errors.json)
 set(LINT_SUPPRESS_URL_BASE "https://raw.githubusercontent.com/neovim/doc/gh-pages/reports/clint")
 set(LINT_SUPPRESS_URL "${LINT_SUPPRESS_URL_BASE}/errors.json")
 set(LINT_PRG ${PROJECT_SOURCE_DIR}/src/clint.py)
+set(CLANG_TIDY_CONFIG_FILE ${PROJECT_SOURCE_DIR}/.clang-tidy)
```

Comment:
doesn't clang-tidy find this implicitly? that was the reason for putting it in the root dir instead of a subdir like config/

Reply:
Oh I forgot to point it to the build directory. I also removed the step with linking the database. I'll fix it ASAP.